src/_support/Page/Acceptance/Administrator/MenuItemFormPage.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @package  Joomla.Test
+ * @subpackage  AcceptanceTester.Page
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc.All rights reserved.
+ * @license     GNU General Public License version 2 or later;see LICENSE.txt
+ */
+namespace Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Page object class to menuitem form page.
+ *
+ * @category  Users
+ * @package   Page\Acceptance\Administrator
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @since     4.0.0
+ */
+class MenuItemFormPage
+{
+	/**
+	 * Menu Item Title
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $menuItemTitle = '#jform_title';
+
+	/**
+	 * Menu Item Alias
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $menuItemAlias = '#jform_alias';
+
+	/**
+	 * Menu Item Save
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static  $saveButton = '#toolbar-apply';
```

Comment:
Is this used anywhere? You use the toolbar click method in the test?!

Reply:
I forgot to remove that. Thanks